src/TStringUtils.cpp

```diff
@@ -24,14 +24,15 @@ bool TStringUtils::isQuote(QChar ch)
     return isOneOf(ch, qsl("\'\""));
 }
 
-bool TStringUtils::isOneOf(QChar ch, const QString& chars)
+bool TStringUtils::isOneOf(QChar inputCharacter, const QString& characterSet)
```

Comment:
I wonder when this is used, if the coder has considered that not all graphemes can be contain within **one** `QChar`! 😲 

Just an observation, no action needed for this PR.